src/linter/lazy_test.go

```diff
@@ -13,6 +13,16 @@ import (
 
 var once sync.Once
 
+func filterReports(name string, reports []*Report) []*Report {
+	out := reports[:0]
```

Comment:
This code modifies the original slice which is not documented and not expected here. I do not think we should be too stingy about extra memory allocations, especially in tests.